code/modules/organs/subtypes/machine.dm

```diff
@@ -138,14 +138,68 @@
 	name = "identification tag"
 	organ_tag = "ipc tag"
 	parent_organ = BP_HEAD
-	icon = 'icons/obj/telescience.dmi'
-	icon_state = "gps-c"
-	dead_icon = "gps-c"
+	icon = 'icons/obj/ipc_utilities.dmi'
+	icon_state = "ipc_tag"
+	item_state = "ipc_tag"
+	dead_icon = "ipc_tag_dead"
+	contained_sprite = TRUE
+	var/auto_generate = TRUE
+	var/serial_number = ""
+	var/ownership_info = IPC_OWNERSHIP_COMPANY
+	var/citizenship_info = CITIZENSHIP_BIESEL
 
 /obj/item/organ/internal/ipc_tag/Initialize()
 	robotize()
 	. = ..()
 
+/obj/item/organ/internal/ipc_tag/examine(mob/user)
+	..()
+	to_chat(user, SPAN_NOTICE("Serial Autogeneration: [auto_generate ? "Yes" : "No"]"))
+	to_chat(user, SPAN_NOTICE("Serial Number: [serial_number]"))
+	to_chat(user, SPAN_NOTICE("Ownership Info: [ownership_info]"))
+	to_chat(user, SPAN_NOTICE("Citizenship Info: [citizenship_info]"))
+
+/obj/item/organ/internal/ipc_tag/attackby(obj/item/W, mob/user)
+	if(W.ismultitool())
```

Comment:
This might be a tad too easy and should probs require something more elaborate (i.e. emag, specialized tool from robotics, modified tag analyzer, ...)

Reply:
Lore team wants it easy to do, but highly illegal to do so. The hard part is not doing it, the hard part is getting away with it.

Reply:
Discussed with the deputy loremaster, the outlined changes are alright with them.
Please require a hacked version of the ipc tag scanner to modify the tag.
(Hacking the tag scanner should be similar to hacking a airlock)